cli/context_command.go

```diff
@@ -174,7 +174,15 @@ func (c *ctxCommand) editCommand(pc *fisk.ParseContext) error {
 		return err
 	}
 
-	return c.showCommand(pc)
+	// There was an error with some data in the modified config
+	// Save the known clean version and show the error
+	err = c.showCommand(pc)
+	if err != nil {
+		ctx.Save(c.name)
+		return fmt.Errorf("updated context validation failed - rolling back changes\n%w", err)
```

Comment:
cool, sorry to be annoying lets just do go standard way so `"updated context validation failed - rolling back changes: %w"` then its all good to do, do that wait for green and you can merge when ready (if you can, dont know what permission you have yet)

Reply:
Yep, no worries - I added the newline for readability since there is a lot of output embedded in that error return.

Reply:
hmm, ok, then perhaps better as it was lol - I will play with it and tweak the UX after no worries, thanks for the fix!